modules/pigz/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://zlib.net/pigz/",
+    "maintainers": [
+        {
+            "email": "alex@aspect.dev",
+            "github": "alexeagle",
+            "name": "Alex Eagle"
+        }
+    ],
+    "repository": [],
```

Comment:
Note, I can't link to the github repo here because we have a presubmit check

> BcrValidationResult.FAILED: The source URL of pigz@2.7 (https://zlib.net/pigz/pigz-2.7.tar.gz) doesn't match any of the module's source repositories ['github.com:madler/pigz'].

And there's no tag I can apply on the PR to skip that like we have for the stable URL check. @meteorcloudy is that easy to add?

Reply:
Yeah, it should be pretty easy. 

Sent https://github.com/bazelbuild/bazel-central-registry/pull/721 and https://github.com/bazelbuild/continuous-integration/pull/1704

Added label "skip-source-repo-check".

Feel free to follow this pattern to customize more checks if necessary.